_includes/v22.1/sidebar-data/migrate.json

```diff
@@ -9,82 +9,75 @@
       ]
     },
     {
-      "title": "Migrate from Serverless to Dedicated",
-      "urls": [
-        "/cockroachcloud/migrate-from-serverless-to-dedicated.html"
-      ]
-    },
-    {
-      "title": "Migrate from Oracle",
+      "title": "Migrations Page",
       "urls": [
-        "/${VERSION}/migrate-from-oracle.html"
+        "/cockroachcloud/migrations-page.html"
       ]
     },
     {
-      "title": "Migrate from PostgreSQL",
```

Comment:
Were these intentionally removed? Or am I completely missing where they moved to?

Reply:
Yes, I intentionally hid the link to this, MySQL, and Oracle in this PR because the steps in them are actually not correct/functional and it's arguably worse to have them visible until they are fixed (sooner than later).

Reply:
Ahhh gotcha! Makes sense 👍